# process setting
enable_rectified: false
enable_disparity: false
enable_depth: false
enable_point: false
enable_detector: false

# device setting
# resolutuin index: 1 = IMG_640(640X400), 2 = IMG_1280(1280X800)
resolution_index: 1
# image frame rate range: IMG_640: 25, 50, 100, 200. IMG_1280: 25 50 100
framerate: 50
# imu frequency range: 100, 200, 500, 1000
imu_frequency: 1000